<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanWindow</class>
 <widget class="QMainWindow" name="ScanWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>757</width>
    <height>423</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>757</width>
    <height>423</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>757</width>
    <height>423</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Scan for viruses</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/clamguard/Gerald-G-Clam-Security-Guard-ico32x32.png</normaloff>:/clamguard/Gerald-G-Clam-Security-Guard-ico32x32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txtScan">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>741</width>
      <height>341</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblOutput">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>54</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCancelScan">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>380</y>
      <width>121</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel Scan</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClose">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>380</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resourses.qrc"/>
 </resources>
 <connections/>
</ui>
